jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/RabitTracker.java

```diff
@@ -16,45 +17,40 @@
   // Maybe per tracker logger?
   private static final Log logger = LogFactory.getLog(RabitTracker.class);
   // tracker python file.
-  private static File tracker_py = null;
+  private static String tracker_py = null;
   // environment variable to be pased.
   private Map<String, String> envs = new HashMap<String, String>();
   // number of workers to be submitted.
   private int num_workers;
-  // child process
-  private  Process process = null;
-  // logger thread
-  private Thread logger_thread = null;
-
-  //load native library
```

Comment:
This cannot be deleted, mainly because we need to call trackerPy to create temp file to the python in resource.